contracts/src/v0.7/dev/KeeperRegistryDev.sol

```diff
@@ -41,7 +41,6 @@ contract KeeperRegistryDev is
   address private constant IGNORE_ADDRESS = 0xFFfFfFffFFfffFFfFFfFFFFFffFFFffffFfFFFfF;
   bytes4 private constant CHECK_SELECTOR = KeeperCompatibleInterface.checkUpkeep.selector;
   bytes4 private constant PERFORM_SELECTOR = KeeperCompatibleInterface.performUpkeep.selector;
-  uint256 private constant CALL_GAS_MAX = 5_000_000;
   uint256 private constant CALL_GAS_MIN = 2_300;
```

Comment:
can we rename this variable? I always found it kinda confusing. `PERFORM_GAS_MIN` would be my vote.